src/foundry/common/prosemirror/_module.d.mts

```diff
@@ -31,7 +36,6 @@ declare const dom: {
 declare const defaultPlugins: Record<
   | "inputRules"
   | "keyMaps"
-  | "images"
```

Comment:
Interesting. Foundry removed this plugin?

Reply:
At some point?  It wasn't in there as of v11.315 either...